README.md

```diff
@@ -1,15 +1,15 @@
 html-metadata
 =============
-
-# MetaData html scraper and parser for Node.js (supports Promises and callback style)
+[![npm](https://img.shields.io/npm/v/html-metadata.svg)](https://www.npmjs.com/package/html-metadata)
+> MetaData html scraper and parser for Node.js (supports Promises and callback style)
```

Comment:
Is the carrot intentional? It puts it in a quote, looks weird to me. :)

Reply:
Yep! Intentional, to break up the title and the explanation bit. It's totally not related to the change I made, I'm more than happy to revert it back.